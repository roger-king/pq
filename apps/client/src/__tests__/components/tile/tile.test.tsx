import React from 'react';
import renderer from 'react-test-renderer';

import Tile from '../../../components/tile';

describe('<Tile />', () => {
  test('should display Tile without background', async () => {
    const tree = renderer.create(<Tile />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('should display Tile with background', async () => {
    const tree = renderer.create(<Tile background="#000" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
