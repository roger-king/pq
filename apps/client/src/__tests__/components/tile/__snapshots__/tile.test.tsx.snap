// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tile /> should display Tile with background 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kShffx"
/>
`;

exports[`<Tile /> should display Tile without background 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 gCllZT"
/>
`;
